content/tracing/advanced_usage/_index.md

```diff
@@ -1629,6 +1709,40 @@ Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
 
 [1]: /help
 {{% /tab %}}
+{{% tab "PHP" %}}
+
+The following example injects the required `dd.trace_id` and `dd.span_id` identifiers into the log message to associate the log entry with the current trace.
```

Comment:
Just out of curiosity, is this tested (at least manually)? In any case I would specify that this is currently only available for manual logging and in the future will also be available for auto-instrumentation. :)

Reply:
Absolutely! :) I've been testing all the code examples throughout all the docs with a little test app I made. I make way too many type-o's otherwise. Hehe. Here's a trace with linked logs in it:

<img width="998" alt="screen shot 2019-01-25 at 8 04 16 am" src="https://user-images.githubusercontent.com/578780/51747636-1b700000-2078-11e9-9830-5c2cbe926068.png">

As far as mentioning that there will be a feature coming soon, I didn't  include that since the docs style guide specifies [not to mention future features](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#donts).

> Don't subtly hint at the past or future state of features/support [...] Avoid temporal words like "currently", "now", etc. Just describe how things are

Do we need to make an exception for the PHP tracer since it's pre-GA? :)

Reply:
No No no exception 😄